cardinal/types/debug.go

```diff
@@ -0,0 +1,12 @@
+package types
+
+import "encoding/json"
+
+type DebugStateRequest struct{}
```

Comment:
Request and Response type is something that is specific to server. It's not supposed to be a common type that is passed around through different packages, so I don't see a reason why this should be here.

Reply:
Right now it's used in cardinal and server. I'll look into consolidating it.

Reply:
nm, just moving it ended up working.